modules/nextrollIdSystem.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * This module adds Nextroll ID to the User ID module
+ * The {@link module:modules/userId} module is required
+ * @module modules/nextrollIdSystem
+ * @requires module:modules/userId
+ */
+
+import { deepAccess } from '../src/utils.js';
+import { submodule } from '../src/hook.js';
+import { getStorageManager } from '../src/storageManager.js';
+
+const NEXTROLL_ID_LS_KEY = 'dca0.com';
+const KEY_PREFIX = 'AdID:'
+
+export const storage = getStorageManager();
+
+/** @type {Submodule} */
+export const nextrollIdSubmodule = {
+  /**
+   * used to link submodule with config
+   * @type {string}
+   */
+  name: 'nextroll',
```

Comment:
this should be `nextrollId`

Reply:
Documentation mentioned it was recommended to use the `Id` suffix, but unfortunately our Adapter module is currently using the `nextroll`  name.  We'd like to keep this name if possible, what's your opinion on this?